controllers/auth.go

```diff
@@ -49,7 +50,30 @@ func HandleRegister(w http.ResponseWriter, r *http.Request) {
 
 // GetProfile takes the UserForm, edits the profile
 func GetProfile(w http.ResponseWriter, r *http.Request) {
-	// do stuff
+	var user models.User
+	err := json.NewDecoder(r.Body).Decode(&user)
+	if err != nil {
+		panic(err)
+	}
+
+	db, err := config.InitDB()
+	if err != nil {
+		panic(err)
```

Comment:
Replace panic() by an error handling:
- log the error in the console with an understandable message 
- set the error code in the header (here, it should be 500 since the error comes from the server)